HACKING.md

```diff
@@ -0,0 +1,30 @@
+# Hacking on Docs
+
+Install dependencies:
+
+```bash
+sudo apt install python3-sphinx
+```
+
+To build the html documentation:
+
+```bash
+make docs
+```
+
+To browse the html documentation locally:
+
+```bash
+make docs
+cd docs/_build/html
+python -m SimpleHTTPServer 8765
```

Comment:
You don't actually need to run a server; just pointing to the static HTML files using a `file://` URL works fine.  I usually use `gnome-open docs/_build/html/index.html`

Reply:
Thats a lot easier!

`gnome-open` isn't installed by default in 17.10, but `xdg-open` is, so I'll use that.